pycontw/pycon.tw
# feat(remote-label): display label on schedule if it's given from remote

## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies


## Description
A field of `ProposedTalkEvent` named `is_remote` is add. If a talk event is assigned as remote, a label will be displayed on the schedule page (after a new version of the schedule is generated).

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to the admin page and select specific talks as remote (check the tickbox and save).
2. Go to `<host>/<lang>/conference/schedule/new/`

## Expected behavior
The label indicating the talk is given from remote will be displayed.

## Related Issue
#908 

## More Information

![image](https://user-images.githubusercontent.com/24987826/92114911-33972c00-ee24-11ea-9efa-81cccdfd1913.png)


## Flaw

- Cannot translate for the text on the label
- If the title of the talk is too long, the label will be hidden
- Alternatives: 
    1. put on an icon along with other icons (e.g. python level) instead of a label?
	2. change the text on the label from `Remote Talk` to `Remote`?
